simplified-viewer-epub-readium2/src/main/java/org/nypl/simplified/viewer/epub/readium2/Reader2Activity.kt

```diff
@@ -505,6 +499,24 @@ class Reader2Activity : AppCompatActivity(R.layout.reader2) {
     }
   }
 
+  private fun onBookmarkWasCreated(
+    possiblyRemoteBookmark: Bookmark?,
+    localBookmark: Bookmark.ReaderBookmark,
+    event: SR2Event.SR2BookmarkEvent.SR2BookmarkCreate
+  ) {
+    val savedBookmark = possiblyRemoteBookmark ?: localBookmark
```

Comment:
Instead of passing the two bookmarks as an argument and to this assignment inside the method, you could just pass one argument with this verification here, i.e., `onBoomarkWasCreated(possiblyRemoteBookmark ?: localBookmark, event)`